### PR TITLE
doc(contribute): add server description for contribute.md

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [x] add necessary doc

### CODE REVIEW

**Review by anda-ren — APPROVED** [review]
lgtm

File: docs/docs/community/contribute.md [hunk 1]
```diff
@@ -94,18 +101,117 @@ swcli, version 0.0.0.dev0
...
+### 5.4 Lint and Test
 
 Run unit test, E2E test, mypy lint, flake lint and isort check in the `starwhale` directory.
 
 ```console
 make client-all-check
 ```
 
-## 5. Development environment for Standalone Instance
+## 6. Development environment for Standalone Instance
```
[the remark this answers is not in the document]
Comment (goldenxinxing) [reply docs/docs/community/contribute.md]: done

**Review by jialeicui — APPROVED** [review]
LGTM

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/community/contribute.md [hunk 2]
```diff
@@ -99,18 +113,129 @@ swcli, version 0.0.0.dev0
...
+- 项目构建工具：Maven
+- 开发框架：Spring Boot+Mybatis
+- 测试框架：Junit5（其中mock框架为mockito，断言部分使用hamcrest，数据库、web服务等模拟使用Testcontainers）
+- 代码检查：使用maven插件 maven-checkstyle-plugin
+
+#### 6.2.1 前置条件
+
+- OS：Linux、macOS或Windows
+- JDK: >=11
+- Docker：>=19.03
```
Comment (anda-ren) [inline docs/i18n/zh/docusaurus-plugin-content-docs/current/community/contribute.md:146]: doesn't match with englisht version which doesn't have docker/minio etc.

Comment (goldenxinxing) [reply docs/i18n/zh/docusaurus-plugin-content-docs/current/community/contribute.md:146]: done

**Review by anda-ren — APPROVED** [review]
lgtm

**Pull request merged**